app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java

```diff
@@ -54,6 +56,23 @@ private Constants(final IServiceLocator serviceLocator) {
         return new ModuleWrapper<>(new Constants(serviceLocator));
     }
 
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getPasswordPath() {
```

Comment:
Why don't the encrypted and non-encrypted version of the database don't have the same filename?